src/containers/SmbFlow/SmbFlow.jsx

```diff
@@ -143,6 +151,19 @@ class SmbFlow extends BaseMobileFlow {
         return Promise.resolve();
       }
 
+      if (err.response.status === 403 && err.response.data.detail === 'User does not have available calculations.') {
+        const {
```

Comment:
Just a suggestion:

make it as const (NO_AVAILABLE_CALCULATIONS_TEXT = 'User does not have available calculations.') e.g.

Reply:
yep, the suggestion is good, but I'd prefer to leave as it is, just to keep curent files states as in a private-mtm-widget repo. What do you think?